### PR TITLE
Use unified image

### PR DESCRIPTION
**Pull request by masayag**
Using the same image for must-gather and node-gather simplifies a lot of
image releasing issues. It allows to use the exact same image to collect
data from nodes as well from the cluster.

### CODE REVIEW

**Comment by pkliczewski** [comment]
@masayag Have you tried to verify it?

**Comment by masayag** [comment]
> @masayag Have you tried to verify it?

yes, seems to work the same as before this patch.

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,15 @@
-FROM quay.io/openshift/origin-must-gather:latest
+FROM quay.io/openshift/origin-must-gather:latest as builder
+
+FROM centos:7
```
Comment (pkliczewski) [inline Dockerfile]: what do we need from centos7?

Comment (masayag) [reply Dockerfile]: That’s the base image for node gather. All requires packages for collecting information are installed from.

**Pull request merged**